README_LEGAL_REFERENCES_TSV.md

```diff
@@ -0,0 +1,52 @@
+# legal_references.tsv
+Lenkeinjectoren bruker denne fila for å legge til riktig referanse til en lov eller forskrift. Det er en tab separert fil med venstre kolonne som inneholder navn på lover og høyre kolonne som inneholder ID'er til de aktuelle dokumentene hos Lovdata (eller evt. hos oss). Disse dokument referansene brukes for å generere en lenke. 
+
+## Struktur
+Både venstre og høyre kolonne kan inneholder flere navn eller dokument referanser separert med vertikal linje `|` som f. eks:
```

Comment:
Ville prefikset dette med 
> De to kolonnene er separert med tab ('	').

for da kan tab-karakteren enkelt copy-pastes om editoren man bruker oversetter trykk på tab-knappen til N spaces.